### ISSUE TITLE
Export of `BlindSign`

### ISSUE THREAD

**Issue by brianorwhatever** [digitalbazaar/bbs-signatures#15]
I found the `BlindSign` method however it doesn't look like there is an interface into this to use it. Is it not quite ready yet or am I missing something?

**Comment by dlongley** [comment]
Yes, it isn't exposed yet as the underlying IETF draft was undergoing significant changes. Those changes are now in and work is being done to update the implementation here and determine an API to expose.

**Comment by brianorwhatever** [comment]
That makes sense. I am still working on the basic implementation so not needed yet but am very much looking forward to playing with it.
